src/community/geofence-server/src/main/java/org/geoserver/geofence/rest/UsersRestController.java

```diff
@@ -0,0 +1,151 @@
+package org.geoserver.geofence.rest;
```

Comment:
@mcrmcr , are you using restful? Cause I made everything with spring as you can see.